djangocms_alias/admin.py

```diff
@@ -173,3 +173,13 @@ def has_module_permission(self, request: HttpRequest) -> bool:
         """Hides admin class in admin site overview"""
 
         return False
+
+    def delete_model(self, request: HttpRequest, obj: AliasContent):
+        if obj.alias.contents.filter(language=obj.language).count() == 1:
```

Comment:
I'm not sure if that condition is correct. If verisoning is stalled, `alias.contents` will give published results only.

```suggestion
        if obj.alias._default_manager.filter(language=obj.language).count() == 1:
```